src/line-chart.coffee

```diff
@@ -9,6 +9,7 @@ directive('linechart', ['n3utils', '$window', '$timeout', (n3utils, $window, $ti
   link  = (scope, element, attrs, ctrl) ->
     _u = n3utils
     dim = _u.getDefaultMargins()
+    dispatch = _u.getEventDispatcher()
```

Comment:
Just out of curiosity, is there any specific reason you didn't use the `initialHandlers` object ?